pages/linux/autorandr.md

```diff
@@ -11,7 +11,11 @@
 
 `autorandr`
 
-- Change the profile:
+- Load the first detected profile:
+
+`autorandr -c`
+
+- Load a specific profile:
 
 `autorandr -l {{profile_name}}`
```

Comment:
per contributing guideline #3, I figure it's better to use the word "load" for this example since that is the spelled out version of the single-letter option. It's even more essential now that I'm adding the actual `--change` as another example.